doc/modules/feature_extraction.rst

```diff
@@ -101,6 +101,81 @@ memory the ``DictVectorizer`` class uses a ``scipy.sparse`` matrix by
 default instead of a ``numpy.ndarray``.
 
 
+.. _column_transformer:
+
+ColumnTransformer for heterogeneous data
+========================================
+
+.. warning::
+
+    The :class:`experimental.ColumnTransformer <sklearn.experimental.ColumnTransformer>`
+    class is experimental and the API is subject to change.
+
+Many datasets contain features of different types, say text, floats, and dates,
+where each type of feature requires separate preprocessing or feature extraction steps.
+Often it is easiest to preprocess data before applying scikit-learn methods,
+for example using `pandas <http://pandas.pydata.org/>`__.
```

Comment:
I think I corrected it but the comment was not on the right line. So it still appear but don't take it into account.